e2e/test/scenarios/filters-reproductions/dashboard-filters-reproductions.cy.spec.js

```diff
@@ -548,7 +547,6 @@ describe("issues 15119 and 16112", () => {
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
     cy.findByText(reviewerFilter.name).click();
     popover().contains("adam").click();
-    multiAutocompleteInput().blur();
```

Comment:
Reverts this
https://github.com/metabase/metabase/pull/43293/files#diff-9db584711720ed7f09df046944b2c4e4a71ba62532dccb03c89930cffaa9e779R550